TACC/Core-Styles
# feat: style read-only fields

## Overview

Fields that are `readonly` should appear as such.

## Related

- required by https://github.com/tapis-project/authenticator/pull/83

## Changes

- **added** styles for `readonly` fields

## Testing

1. Load a form demo.
2. Add `readonly` attribute to a field.
3. Verify border is removed.

## UI

| normal | `readonly` |
| - | - |
| <img width="1194" alt="before" src="https://github.com/user-attachments/assets/47bdbac0-0f6f-46e2-a525-bccadc5fe085"> | <img width="1194" alt="after" src="https://github.com/user-attachments/assets/e11dc578-4bb5-4e4e-82e9-028b69c22187"> |
